render_delegate/hdarnold.h

```diff
@@ -32,6 +32,6 @@
 #define USD_DO_NOT_BLIT
 #endif
 
-#if AI_VERSION_NUMBER > 60201
-#define AI_MULTIPLE_RENDER_SESSIONS
+#if ARNOLD_VERSION_NUMBER > 60201
```

Comment:
You can use the readily available `ARNOLD_VERSION_NUM` defined in `ai_version.h`.